test/membrane_rtc_engine/hls_endpoint_test.exs

```diff
@@ -73,14 +62,12 @@ defmodule Membrane.RTC.HLSEndpointTest do
       Engine.message_endpoint(rtc_engine, file_endpoint_id, :start)
 
       assert_receive({:playlist_playable, :video, ^output_dir}, 10_000)
+      assert_receive({:segment, "video_segment_1" <> _}, 15_000)
+      assert_receive({:manifest, %{video_segments: 2}})
```

Comment:
This could be put into the function and then call it in all tests.
```elixir
 defp assert_track(type) do 
      assert_receive({:playlist_playable, :"#{type}", ^output_dir}, 10_000)
      assert_receive({:segment, "#{type}_segment_1" <> _}, 15_000)
      assert_receive({:manifest, %{"#{type}_segments": 2}})
 end
```